Tasks/GradleV2/Tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Agent.Tasks.Gradle",
+  "version": "2.139.1",
+  "author": "Microsoft Corporation",
+  "description": "Build with Gradle",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
+  },
+  "dependencies": {
+    "vsts-task-lib": "2.6.0",
+    "@types/mocha": "5.2.5",
+    "mocha": "5.2.0"
```

Comment:
This should not be needed, my understanding was vsts-task-lib should bring mocha and each task shouldn't have to add it as an explicit dependency